client/login/wp-login/index.jsx

```diff
@@ -36,7 +36,7 @@ import RequestLoginEmailForm from '../magic-login/request-login-email-form';
 import { recordTracksEvent } from 'state/analytics/actions';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
 
-class Login extends React.Component {
+export class Login extends React.Component {
```

Comment:
Note that it has a side effect for webpack if we use `require` to load the module.
Since we now have more than a `default` exported, we would need to write `const Login = require( 'login/wp-login' ).default` explicitly.